src/hydro_system.hpp

```diff
@@ -923,11 +924,22 @@ void HydroSystem<problem_t>::ComputeFluxes(amrex::MultiFab &x1Flux_mf, amrex::Mu
 
 		quokka::valarray<double, nvar_> U_L = {sL.rho, sL.rho * sL.u, sL.rho * sL.v, sL.rho * sL.w, sL.E, sL.Eint};
 		quokka::valarray<double, nvar_> U_R = {sR.rho, sR.rho * sR.u, sR.rho * sR.v, sR.rho * sR.w, sR.E, sR.Eint};
+
+		// conserve flux of mass scalars
```

Comment:
Can you add a reference to the CMA paper here? This will make it clear to future readers of the code that's the algorithm that you've implemented to do this.